tests/test_crs/valid_uperf_store.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: ripsaw
 spec:
   elasticsearch:
-    server: elk-b09-h30-r720xd.rdu.openstack.engineering.redhat.com
+    server: <es_server>
```

Comment:
Shouldn't we template this into a variable instead of just having a placeholder in the yaml?